src/block-management/block-holder.js

```diff
@@ -20,16 +28,25 @@ import { BlockEdit } from '@wordpress/editor';
 import TextInputState from 'react-native/lib/TextInputState';
 
 type PropsType = BlockType & {
+	clientId: string,
+	rootClientId: string,
 	isSelected: boolean,
+	isFirstBlock: boolean,
+	isLastBlock: boolean,
 	showTitle: boolean,
-	onChange: ( clientId: string, attributes: mixed ) => void,
+	getBlockIndex: ( clientId: string, rootClientId: string ) => number,
+	getPreviousBlockClientId: ( clientId: string ) => string,
+	getNextBlockClientId: ( clientId: string ) => string,
+	onChange: ( attributes: mixed ) => void,
 	onReplace: ( blocks: Array<Object> ) => void,
```

Comment:
I think `onReplace()` is not passed in props anymore so, let's remove it from the propTypes as well.